.github/workflows/node.js.yml

```diff
@@ -9,29 +9,27 @@ on:
 
 jobs:
   build:
-
-    runs-on: macos-latest
-
+    name: Rebuild Manifest
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [12.x]
+        node-version: [18.x]
         # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
-
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
       with:
         ref: ${{github.event.pull_request.head.ref}}
         repository: ${{github.event.pull_request.head.repo.full_name}}
     - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v2
+      uses: actions/setup-node@v4
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm ci
     - run: npm run build
     - run: git config --local user.name  ${{ github.actor }}
     - run: git add index.json
     - run: git commit -m "Automatic manifest update"
-    - name: 
+    - name: Push to GitHub
       uses: ad-m/github-push-action@master
       with:
         github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This is also a line off, but maybe the reason there's a local branch with the manifest change but it isn't in the PR from the fork is because the action is pushing `${{github.event.pull_request.head.ref}}`? Is that maybe not pointing to a forked branch? I haven't actually looked into this enough yet to confirm.

Reply:
I think you're right -- and based on our offline discussion, I think we both spent some time trying to understand this and find some clear documentation about how to push back to a forked branch -- and couldn't find it. So for now, I think we leave it as-is, and if we get a PR from a forked branch without the updated manifest, we should ask the submitter to please run `npm run build` and push the resulting change.